packages/form-layout/src/vaadin-form-item-mixin.js

```diff
@@ -42,6 +42,16 @@ export const FormItemMixin = (superClass) =>
       this.__fieldNode = null;
     }
 
+    connectedCallback() {
+      super.connectedCallback();
+      const spacing = getComputedStyle(this).getPropertyValue('--vaadin-form-item-row-spacing');
+      if (spacing !== '' && parseInt(spacing) !== 0) {
+        console.warn(
+          '`--vaadin-form-item-row-spacing` is deprecated since 24.7. Use `--vaadin-form-layout-row-spacing` instead.',
```

Comment:
Are you planning to introduce `--vaadin-form-layout-row-spacing` in a separate PR?

Reply:
I wasn't sure at the beginning, but I decided to add the new CSS variable here, too. PTAL